packages/admin-ui/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "compilerOptions": {
     "lib": [
-      "ES7",
-      "DOM"
+      "es7",
+      "es2017",
+      "dom"
```

Comment:
why did this need changing? I took the values from https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
Sorry I should have commented. Es2017 is needed for ```Object.values```.

Reply:
For the lowercase versions ... webstorm complained